newIDE/app/src/Utils/Window.js

```diff
@@ -342,6 +342,10 @@ export default class Window {
     window.open(url, '_blank');
   }
 
+  static getOrientation(): 'portrait' | 'landscape' {
+    return window.screen.orientation.type.split('-')[0];
```

Comment:
should we have a try catch around this?
I'm scare with using window, when in some weird cases it's not available. (maybe not useful)

Reply:
Good idea. I'll add it!